src/tools/pub/cmd/index.md

```diff
@@ -143,14 +143,17 @@ to make it directly available from the command line.
 Several command-line options work with all of the pub subcommands.
 These include:
 
-`--help` or `-h`
-: Print usage information.
+### `--help` or `-h`
 
-`--trace`
-: Print debugging information when an error occurs.
+Print usage information.
```

Comment:
We should change these to start with verbs ending in "s", like you did elsewhere.